code-challenges.js

```diff
@@ -7,33 +7,90 @@
 
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
-// Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
-
+// Create the code and test each of the variables provided. Expected output: 42 is below boiling point, 350 is above boiling point, 212 is at boiling point
 var temp = 42
-// var temp = 350
+//var temp = 350
 // var temp = 212
 
+if(temp <= 42){
+console.log("temp is below boiling point")
+} else if(temp >= 350){
+console.log("above boiling point")
+} else("temp = 212"){
+console.log(temp, "at boiling point");
+}
+
```

Comment:
Looks like you are on the right track. The idea here is that you are comparing any given value to boiling point. So the comparisons should all be to 212.